src/quantum/setup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.11.2906.1'
```

Comment:
Will we have a way of injecting this version number with QDK builds, or should it not be kept in sync with QDK versions?

Reply:
Not easily. This gets build as part of the cli-extension repo itself. I guess we could always try to update it as part of the QDK release process, but not high priority imho.